src/Products/Editor/Controllers/EditorApiController.cs

```diff
@@ -524,6 +639,8 @@ private LoadDocumentEntity LoadDocument(string guid, string password)
                     editOptions.EnablePagination = true;
                 }
 
+                IDocumentInfo documentInfo = editor.GetDocumentInfo(password);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'documentInfo'.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=6003709)